### ISSUE TITLE
Not able to generate a MicroQRCode

### ISSUE THREAD

**Issue by cherryatdusty** [nu-book/zxing-cpp#380]
I might be missing the steps here but I haven't found a way to generate a Micro QR Code. I believe the reader works with Micro.

**Comment by duanqn** [comment]
Reading https://github.com/nu-book/zxing-cpp/issues/319, I believe the writer part does not exist for MicroQR.

**Comment by axxel** [comment]
That is correct. See also #332. If you need that right now, I'd recommend to use `zint`.

**Comment by axxel** [comment]
For what its worth I added a comment to the README making explicit which codes are currently not supported in the writers.

**Comment by KrasnayaPloshchad** [comment]
As this is standardized in ISO/IEC 23941:2022, it’s reasonable to make implementation

**Comment by axxel** [comment]
The 2.3.0 release allows to configure the library with `cmake -DZXING_WRITERS=NEW -DZING_EXPERIMEMTAL_API=ON` and then this symbology can be written, e.g. with
```c++
#include "WriteBarcode.h"
auto svg = WriteBarcodeToSVG(CreateBarcodeFromText("hello", BarcodeFormat::MicroQRCode));
```
